tools/defaults.bzl

```diff
@@ -97,7 +92,7 @@ def ng_module(
 
     _ng_module(
         srcs = srcs,
-        type_check = _ENABLE_NG_TYPE_CHECKING,
+        type_check = True,
```

Comment:
We can remove this, since it's enabled by default.

Reply:
Done